build-a-test-app.md

```diff
@@ -9,6 +9,33 @@ toc: false
 
 ## Python
 
+~~~ python
+# Import the driver.
+import psycopg2
+
+# Assuming a "bank" database already exists, connect to it.
+conn = psycopg2.connect(database='bank', user='root', host='ROACHs-MacBook-Pro.local', port=26257)
```

Comment:
I'd rather use a non-root user here. Since this is insecure mode, you should just need to grant the appropriate permissions when creating the database (which I assume will be documented in the CLI section before the per-language breakout) and change the name here.

`host` should be `'localhost'` since that's the most likely scenario. (although be aware that some mysql drivers interpret the hostname "localhost" specially in a way that wouldn't work for us. I don't think any postgres drivers do the same thing, but it's something to watch out for when you're testing)